lib/elixir/lib/io/ansi.ex

```diff
@@ -29,6 +29,12 @@ defmodule IO.ANSI do
   @typep ansilist :: maybe_improper_list(char() | ansicode() | binary() | ansilist(), binary() | ansicode() | [])
   @type  ansidata :: ansilist() | ansicode() | binary()
 
+  defmacro valid_rgb216(r, g, b) do
```

Comment:
Maybe we should move this check into `color` and `color_background` definitions?
It is simple enough to be defined directly in the user code, furthermore, people will need to `require IO.ANSI`.